inbc-program/README.md

```diff
@@ -100,11 +100,19 @@ System update flow can be broken into two parts:
 1.  Pre-reboot: The pre-boot part is when a system update is triggered.
 2.  Post-reboot: The post-boot checks the health of critical manageability services and takes corrective action.
 
+SOTA on Ubuntu is supported in 3 modes:
+1. Update/Full - Performs the software update.
+2. No download - Retrieves and installs packages.
+3. Download only - Retrieve packages (will not unpack or install).
+
+
 ### Usage
 ```
 inbc sota {--uri, -u=URI} 
-   [--releasedata, -r RELEASE_DATE; default="2026-12-31"] 
-   [--username, -un USERNAME] 
+   [--releasedate, -r RELEASE_DATE; default="2026-12-31"] 
+   [--username, -un USERNAME]
+   [--command, -c COMMAND; default="update"]
```

Comment:
since we are removing --command as an INBC parameter.  We should remove it from the documentation in line 114

Reply:
fixed